chemistrylab/extract_algorithms/extractions/methyl_red.py

```diff
@@ -113,7 +115,7 @@ def reset(self, extraction_vessel):
 
         ethyl = material.EthylAcetate
```

Comment:
Would it make more sense to make this an object rather than doing it multiple times later?